test/cases/image_tests.sh

```diff
@@ -98,7 +98,7 @@ cd $WORKING_DIRECTORY
 
 # Workaround the problem that 'image-info' can not read SELinux labels unknown to the host from the image
 OSBUILD_LABEL=$(matchpathcon -n "$(which osbuild)")
-sudo chcon "$OSBUILD_LABEL" /usr/libexec/osbuild-composer-test/image-info
+sudo chcon "$OSBUILD_LABEL" osbuild-image-info
```

Comment:
Sorry, I missed it, but calling `chcon` is no longer needed.

Reply:
Dropped it in filesystm.sh but missed this one :/

Reply:
Fixed.